spec/features/users/profile/show_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'As a User' do 
+    describe 'When I visit my profile' do
+        it 'By first logging in on the welcome page' do 
+            user = create(:user)
+            user.confirm
```

Comment:
I think you mentioned that this is a part of Devise, that's great! And makes testing user behavior a lot easier.